### PR TITLE
Add a README file to the repo

### PR DESCRIPTION
**Pull request by ShristiGup**
Adding a README file that was missing from the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Rearrange
+=========
+
+This repo is used to rearrange the provided names.
+Turning "FirstName,LastName" into "LastName,FirstName"
```
Comment (RishabhGoswami) [inline README.md]: You should end the line with a period.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Example
```
Comment (RishabhGoswami) [inline README.md]: Should add one more # to make it a smaller heading.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+`rearrange("Clark","Michael")` returns `Michael Clark`
```
Comment (RishabhGoswami) [inline README.md]: It would be nice if you could provide some more examples.

**Review by RishabhGoswami — CHANGES_REQUESTED** [review]
Please address to all comments!!!

**Comment by ShristiGup** [comment]
All changes resolved,plz take another look!!!

**Pull request merged**